### ISSUE TITLE
How the orthogonal projection is done in this work?

### ISSUE THREAD

**Issue by arif2127** [jiajunwu/marrnet#11]
To calculate the projected depth loss and projected surface normal loss, orthogonal projection is done (mentioned in 3.3). If so, then what is the shape of estimated depth and normal (2.5 sketch)? if the shape is not 128x128, then how orthogonal projection done from 128x128x128 voxel ?

**Comment by 1p1pd** [comment]
The 2.5D sketches are downsampled to 128 x 128 for the reprojection loss.

**Comment by arif2127** [comment]
Thanks for your quick response. 
If we calculate the loss function from orthogonal projection, then the model will contain perspective error. is there any other calculation in this work to minimize that error?

**Comment by 1p1pd** [comment]
Our 2.5D sketches are aligned with 3D voxels in section 3.2. There is no such error in reprojection loss.

**Comment by arif2127** [comment]
I have observed that training samples provided from Shapenet dataset is perfectly aligned with 3D voxels and inputs (depth and normal) is rendered with perspective projection from that aligned 3D voxels. If depth projection loss is calculated using orthographic projection where the input depth rendered with perspective projection, then 2D position of input depth can be different with respect to 3D position in voxels. is the input(depth and normal) rendered with orthographic projection from 3D voxels in training dataset?
